src/main/scala/ch/openolitor/stammdaten/repositories/StammdatenRepositoryQueries.scala

```diff
@@ -2656,4 +2656,11 @@ trait StammdatenRepositoryQueries extends LazyLogging with StammdatenDBMappings
       }).list
   }
 
+  protected def getAbweisenheitByLieferungQuery(lieferungId: LieferungId) = {
+    withSQL[Lieferplanung] {
```

Comment:
This is a copy/paste error, right? This can only be `withSQL {`

Reply:
fixed